app/forms/form.rb

```diff
@@ -15,6 +15,10 @@ def initialize(claim:, journey:, params:)
     super
   end
 
+  def persisted?
+    true
+  end
```

Comment:
This could be delegated to `claim`, an instance of `CurrentClaim` that defines `#persisted?`